SQL/database_changelog.txt

```diff
@@ -1,13 +1,47 @@
 Any time you make a change to the schema files, remember to increment the database schema version. Generally increment the minor number, major should be reserved for significant changes to the schema. Both values go up to 255.
 
-The latest database version is 4.0; The query to update the schema revision table is:
+The latest database version is 4.1; The query to update the schema revision table is:
 
-INSERT INTO `schema_revision` (`major`, `minor`) VALUES (4, 0);
+INSERT INTO `schema_revision` (`major`, `minor`) VALUES (4, 1);
 or
-INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (4, 0);
+INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (4, 1);
 
 In any query remember to add a prefix to the table names if you use one.
 
+----------------------------------------------------
+
+3 February 2018, by Jordie0608
```

Comment:
I'd like to see us start to add version numbers in this header, as otherwise if you are behind a few updates you can't know what updates to apply.

Reply:
I'll just do it for all the headers since versioning.